app/controllers/users_controller.rb

```diff
@@ -0,0 +1,4 @@
+class UsersController < ApplicationController
```

Comment:
`rails generate controller Users new` va da xoa .coffee, .scss, .helper. Tien the thi em hoi comment out gem coffee-rails trong gemfile duoc ko a ?